testsuite/running-tests.md

```diff
@@ -54,42 +54,31 @@ This document will enumerate everything you need to do run the sonobuoy based co
 <br/>
 
 ## Running the Arc enabled Data Services Tests (Indirect Mode)
+1. By default, the test suite will run only the Arc enabled Kubernetes tests.
+2. To run the data services tests in [indirect mode](), set the `azure-arc-ds-connect-platform.enable` parameter to true and `azure-arc-ds-connect-platform.CONNECTIVITY_MODE` to indirect in the `azure-arc-conformance.properties` file.
```

Comment:
@v-vegujjula 
The same parameter **azure-arc-ds-connect-platform.enable** needs to be set to true for direct and indirect connect mode

Reply:
Yes this is applicable for both Direct and indirect modes.
It can be disabled if partner wants to run only k8 conformance.